### ISSUE TITLE
Icons Not Showing Up?

### ISSUE THREAD

**Issue by DudeMine** [Pizzabelly/EasyRP#64]
the icons aren't showing up, i have them set in both discord and easyrp config, it looks like this: 
![image](https://user-images.githubusercontent.com/39622890/44908906-a87c8a80-ad60-11e8-81b7-13a6b9031f3e.png)

**Comment by ZER045** [comment]
yeah im having the same problem as well, didnt happen with the other 2 programs i made

**Comment by ryukora** [comment]
maybe you put a wrong name on the config or art assets
